Bug 1910157 - Wire up `max_events` to Server Knobs configuration (#2911)
mozilla/glean: docs/user/user/server-knobs/index.md

# Server Knobs: Glean Data Control Plane

Glean provides Server Knobs, a Data [Control Plane] through which Glean runtime settings can be changed remotely including the ability to enable, disable or throttle metrics and pings through a [Nimbus] rollout or experiment.

> Products can use this capability to control "data-traffic", similar to how a network control plane controls "network-traffic".

Reviewer: ```suggestion
Products can use this capability to control "data traffic", similar to how a network control plane controls "network traffic".
```

nit, but I guess either is fine.

> This provides the ability to do the following:

Reviewer: ```suggestion
Server Knobs provides the ability to do the following:
```

(nit)

- Allow runtime changes to data collection without needing to land code and ride release trains.
- Eliminate the need for manual creation and maintenance of feature flags specific to data collection.
- Sampling of measurements from a subset of the population so that we do not collect or ingest more data than is necessary from high traffic areas of an application instrumented with Glean metrics.
- Operational safety through being able to react to high-volume or unwanted data.
- Visibility into sampling and sampling rates for remotely configured metrics.

> ## Contents
> - [Controlling Metrics with Server Knobs]

Reviewer: nit. Usually we leave a newline after headlines, I guess markdown doesn't really care though

```suggestion
## Contents

- [Controlling Metrics with Server Knobs]
```
